apps/www/components/ui/meteorLines.tsx

```diff
@@ -78,7 +78,7 @@ const MeteorLinesAngular = ({
       animationDuration: speed ? `${speed}s` : `${Math.floor(Math.random() * 10 + 2)}s`,
     }));
     setMeteorStyles(styles);
-  }, [number, windowWidth]);
```

Comment:
the new change doesn’t seem to break existing functionality, but it looks like it wasn’t working as expected before. `windowWidth` isn’t updated on window resize—it’s set once by the effect and never changes, leading to issues like the meteor lines staying on the left when resizing from tablet to full HD (see screenshot).

<img width="500" alt="image" src="https://github.com/user-attachments/assets/6271c820-07a6-453b-b836-322504335732">

cc @MichaelUnkey, Git blame shows you last modified this component. I think the core team should decide whether to fix this. It does not seem critical to me.